### ISSUE TITLE
Leiobunum spec. heeft verkeerde rank

### ISSUE THREAD

**Issue by peterdesmet** [inbo/natuurpunt-exoten-occurrences#15]
De rank van `Leiobunum spec.` is gewijzigd van `multispecies` naar `species`:

```
https://waarnemingen.be/species/8751,Leiobunum spec. A,species,Reuzenhooiwagen
```

Dat is verkeerd, we gebruiken steeds `multispecies` voor `spec.` taxa.

**Comment by karingielen** [comment]
![image](https://github.com/user-attachments/assets/677f373c-a30b-4965-9fa3-29c63134a505)
![Reuzenhooiwagen_SpecMultiSpec](https://github.com/user-attachments/assets/5a3c0256-5fc5-4258-8a31-bda2a0c9a04f)

Ok, had een screenshot geplakt, en dan toch ook maar via add files toegevoegd. 
Maar hetzelfde bekom je als je de tekst gaat lezen die bij de link https://waarnemingen.be/species/8751/ wordt vermeld. 
Is een soort die nog niet geïdentificeerd kan worden? Dan toch nog even wachten?

**Comment by karingielen** [comment]
Haha, dus screenshot inkleven lukt... sorry dat het er nu 2x in staat...

**Comment by peterdesmet** [comment]
Ok, geen `multispecies` dan. Maar het lijkt me dan beter dit als `genus` aan te duiden, want een `species` is het ook niet (en dan komt de "soort" mee in de GBIF soortenlijst en zo). Al weet ik natuurlijk niet of aanpassen naar `genus` problemen zal veroorzaken op de site.

**Comment by karingielen** [comment]
Ok, ik weet niet meer of dit ooit al werd aangepast, maar ik vind in de query géén aanpassingen hieromtrent, en deze staat (nog / opnieuw?) als multispecies gedefinieerd, niet als species. 
Is het daarmee voorlopig weer opgelost?

**Comment by peterdesmet** [comment]
Ja, staat niet meer in de recente data, maar https://waarnemingen.be/species/197493/ als `multispecies` gedefinieerd, dus ok.
